scripts/demo_example.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+from __future__ import print_function
+
+import ga4gh.client as client
+import ga4gh.protocol as protocol
+
+httpClient = client.HttpClient("http://server:80/v0.6.e6d6074")
+request = protocol.SearchVariantsRequest()
+request.variantSetIds = ["1kg-phase1"]
+request.referenceName = "2"
+request.start = 33100
+request.end = 34000
+for variant in httpClient.searchVariants(request):
+    print(
+        variant.referenceName, variant.start, variant.end,
+        variant.referenceBases, variant.alternateBases,
+        sep="\t")
```

Comment:
This example is straight from the demo docs, all I added is the shebang line [and the /current alias]. Is it worth fleshing out? If so, would you mind submitting a PR against this branch?
Thanks for the /current alias, that was upsetting me too and then I found #387